client/app/components/composites/PageSelection/PageSelection.js

```diff
@@ -0,0 +1,43 @@
+import r, { div, a } from 'r-dom';
+import { ArrowButton } from '../../elements/RoundButton/RoundButton';
+import { upsertSearchQueryParam } from '../../../utils/url';
+
+import css from './PageSelection.css';
+
+export default function PageSelection({ currentPage, totalPages, location, pageParam }) {
+  const BUTTON_SIZE = '36px';
```

Comment:
This could potentially come from variables

```javascript
import styleVariables from '../../../assets/styles/variables';
const arrowButtonSize = styleVariables['--PageSelection_buttonSize'];
const arrowButtonMargin = styleVariables['--PageSelection_buttonMargin'];
const arrowButtonsMaxWidth = `${arrowButtonSize *2 + arrowButtonMargin}px`; // i.e. 78px
```